MiniApp/Classes/Display/Displayer.swift

```diff
@@ -0,0 +1,9 @@
+internal class Displayer {
+
+    func getMiniAppView(miniAppPath: URL?) -> MiniAppView? {
```

Comment:
I think `miniAppPath` doesn't need to be optional and optionally return MiniAppView. What do you think?

Reply:
Initial idea was to have a non-optional URL passed to displayer, but since we didn't have any code written to check the validity of mini path URL, this was added. The other PR is going take care of these check. Removed now. Thanks for the comment